vaccine_feed_ingest/runners/ct/state/normalize.py

```diff
@@ -37,6 +37,9 @@ def _get_availability(site: dict) -> Optional[schema.Availability]:
     drop_in, appointments = None, None
 
     for field in ("description", "application_process", "hours_of_operation"):
+        if site[field] == None:
```

Comment:
No need for the `== None` you can just do `if site[field]`

```suggestion
        if site[field]:
```